tests/src/test/java/io/snyk/eclipse/plugin/properties/preferences/PreferencesTest.java

```diff
@@ -33,6 +33,7 @@ void test_DefaultPreferences() {
     assertEquals("true", prefs.getPref(Preferences.MANAGE_BINARIES_AUTOMATICALLY));
     assertEquals("true", prefs.getPref(Preferences.MANAGE_BINARIES_AUTOMATICALLY));
     assertEquals("1", prefs.getPref(Preferences.LSP_VERSION));
+    assertEquals("token", prefs.getPref(Preferences.AUTHENTICATION_METHOD));
```

Comment:
This should use the constant that defines the value "token" instead of using the string value "token" directly.